v03_pipeline/lib/methods/sex_check.py

```diff
@@ -0,0 +1,207 @@
+from __future__ import annotations
+import io
+from enum import Enum
+
+import hail as hl
+import matplotlib.pyplot as plt
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+IMPUTE_SEX_ANNOTATIONS = [
+    'is_female',
+    'f_stat',
+    'n_called',
+    'expected_homs',
+    'observed_homs',
+    'sex',
+]
+
+
+class Ploidy(Enum):
+    AMBIGUOUS = 'ambiguous_sex'
+    ANEUPLOIDY = 'sex_aneuploidy'
+    FEMALE = 'XX'
+    MALE = 'XY'
+
+
+def get_contig_cov(
+    mt: hl.MatrixTable,
+    reference_genome: ReferenceGenome,
+    contig: str,
+    call_rate_threshold: float,
+    af_threshold: float = 0.01,
```

Comment:
I removed the `af_field` argument because I think we're going to expect `AF` for our snv_indel callsets.  If that's not the case we can add it back.

Reply:
A second pass here led me to pulling `AF` off the `variant_qc` struct rather than from the vcf itself.  I think that's right?

Reply:
Following up on myself again, `af_field` is necessary even within the minimal test setup we have, so we need it for sure!